packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -489,7 +494,7 @@ const Sidebar: React.FC<RouteComponentProps & SidebarProps> = props => {
           Fluent <span style={gradientTextStyles}>UI</span>
         </Text>
         <VersionDropdown width={props.width} />
-        <CopyToClipboard value={`yarn add ${pkg.name}@${pkg.version}`} timeout={3000}>
+        <CopyToClipboard value={`yarn add ${pkg.name}@${version}`} timeout={3000}>
```

Comment:
why was this change required ?

Reply:
Without this, 0.0.0-nightly will display `yarn add @fluentui/react-northstar@0.52.0`. With this, it displays `yarn add @fluentui/react-northstar@0.0.0-nightly`. Even though it doesn't make much sense, but it's more correct